test/helper/common.bash

```diff
@@ -410,8 +410,9 @@ function _count_files_in_directory_in_container()
   _run_in_container_bash "[[ -d ${DIRECTORY} ]]"
   assert_success
 
-  _run_in_container_bash "find ${DIRECTORY} -maxdepth 1 -type f -printf 'x\n'"
+  _run_in_container_bash "find ${DIRECTORY} -maxdepth 1 -type f"
```

Comment:
I know that the `-printf 'x\n'` is not strictly required, but I read on SO that this way, one can avoid trouble with special charaters in file names. BATS probably already does that, but it really does not hurt either, does it?

Reply:
I was originally getting a failure from the test as I was only guessing one line should have been returned. To get some feedback I added an `assert_output` into the test method, but the results were all `x` lines... Then I saw this addition and removed it so I could see what the test was vs what I expected.

We could revert this, but then the asserts I do afterwards on what lines (files) are expected in the results wouldn't be usable.

Reply:
Ah, I see, we really do check the output. Then this is completely fine!